File: examples/features/connecting/matlab/connect_url.m
% connect_url.m - Connect to a service using a URL

url = 'rr+tcp://localhost:29200?service=reynard';
c = RobotRaconteur.ConnectService(url);
c.say('Hello from connect_url!');
RobotRaconteur.DisconnectService(c);

File: examples/features/connecting/cs/connect_url.cs
// connect_url.cs - Connect to a service using a URL

using System;
using RobotRaconteur;
using System.Threading;
using System.Threading.Tasks;
using System.Linq;

// Initialize the client node
using (var node_setup = new ClientNodeSetup(args))
{
    // Connect to the Reynard service using a URL
    var c = (experimental.reynard_the_robot.Reynard)RobotRaconteurNode.s.ConnectService(
        "rr+tcp://localhost:29200?service=reynard");

    // Say hello
    c.say("Hello from connect_url!");
}
